excel/util.bal

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+isolated function createRequestPath(string[] pathParameters, string workbookIdOrPath, Query? query = ()) 
+returns string|error {
+    string path = EMPTY_STRING;
+    string[] baseParameters = workbookIdOrPath.endsWith(".xlsx") ? [ME, DRIVE, ROOT + COLON, workbookIdOrPath + COLON, 
+    WORKBOOK] : [ME, DRIVE, ITEMS, workbookIdOrPath, WORKBOOK];
+
+    path = check createPath(path, baseParameters);
+    path = check createPath(path, pathParameters);
+    path = query is Query ? addQueryParameters(path, query) : path;
+    return path;
+}
+
+isolated function createPath(string currentpath, string[] pathParameters) returns string|error {
+    string path = currentpath;
+    if (pathParameters.length() > 0) {
+        foreach string element in pathParameters {
+            if (!element.startsWith(FORWARD_SLASH)) {
+                path = path + FORWARD_SLASH;
+            }
+            path += element;
+        }
+    }
+    return path;
+}
+
+isolated function addQueryParameters(string path, Query query) returns string {
+    string queryPath = path + QUESTION_MARK;
+    if (query?.count is boolean) {
+        queryPath = queryPath + DOLLAR_SIGN + COUNT + EQUAL_SIGN + query?.count.toString();
+    }
+    if (query?.expand is string) {
+        queryPath = queryPath + DOLLAR_SIGN + EXPAND + EQUAL_SIGN + query?.expand.toString();
+    }
+    if (query?.filter is string) {
+        queryPath = queryPath + DOLLAR_SIGN + FILTER + EQUAL_SIGN + query?.filter.toString();
+    }
+    if (query?.format is string) {
+        queryPath = queryPath + DOLLAR_SIGN + FORMAT + EQUAL_SIGN + query?.count.toString();
```

Comment:
Seems like a typo here. Shouldn't it be `query?.format.toString()`  
https://docs.microsoft.com/en-us/graph/query-parameters#format-parameter

Reply:
Removed this method. Thanks